website/docs/docs/getting-started.md

```diff
@@ -11,6 +11,11 @@ We also assume you use `react-native@60.x.x` and above.
 
 For older versions, visit this [installation guide](https://github.com/wix/react-native-notifications/blob/v2/docs/installation.md).
 
+## Introduction 
+This library supports 2 kinds of motifications. 
+1. Local Notifications: These notifications are sent by an application to the device, on which it is running. The notification never leaves the device, on which the application is running. For more details on SENDING local notification, please, refer to [Local Notifications](https://github.com/wix/react-native-notifications/edit/master/website/docs/docs/localNotifications.md) 
+2. Push Notifications. These notifications are sent by an application to any device through a communication/push server. This library provides support for RECEIVING push notifications. For push notifications to work, you need to (1) register the devices, which can receive notifications, on some server/db, and (2) setup notification listeneres on these devices. To register devices, please refer to [Subscription](https://github.com/wix/react-native-notifications/blob/master/website/docs/docs/subscription.md). To setup notification listeneres, please refer to [Events](https://github.com/wix/react-native-notifications/blob/master/website/docs/docs/notifications-events.md). To send a notification you need to use a communication/push server. SENDING push notifications is NOT part of this library. 
```

Comment:
here i think the word `application` can mislead because it's not sent by this application. I would change to `These notifications are sent to the device`

Reply:
I agree with your change. Please, go ahead with it, or do I need to change the PR and resubmit it? 

These notifications are sent to the device through a communication/push server.